ospf6d/ospf6_top.c

```diff
@@ -220,8 +290,8 @@ void ospf6_delete(struct ospf6 *o)
 
 	QOBJ_UNREG(o);
 
-	ospf6_flush_self_originated_lsas_now();
-	ospf6_disable(ospf6);
+	ospf6_flush_self_originated_lsas_now(o);
+	ospf6_disable(o);
```

Comment:
Here is a memory leak in this function. Need to remove ospf6 from the ospf6 list when freeing this memory.

Reply:
Resolved